src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -31,7 +32,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
 {
     internal partial class SuggestedActionsSourceProvider
     {
-        private partial class SuggestedActionsSource : ForegroundThreadAffinitizedObject, ISuggestedActionsSource3
+        private partial class SuggestedActionsSource : ForegroundThreadAffinitizedObject, ISuggestedActionsSourceExperimental
```

Comment:
@KirillOsenkov @Therzok  would you guys intend to support this in VS4mac?  If so, it would allow roslyn to remove a bunch of the legacy code for ISuggestedActionsSource3

Reply:
@davidwengier @sevoku

Reply:
Note this file is also linked into EditorFeatures.Cocoa so it'll apply to both unless we do something else here.

Reply:
What exactly does support this mean in this context?

Reply:
> Note this file is also linked into EditorFeatures.Cocoa so it'll apply to both unless we do something else here.

Extension to this note: We're currently not inserting any bits from Roslyn main into VS Mac, so merging this could cause a lot of pain when we do start up again (once d17 vs mac has settled down), and cause a bunch of extra work at a really bad time.

Perhaps a better approach for the short term would be to un-link these files, and keep the old versions (or part thereof with as much code sharing as possible) in the Cocoa project.

Reply:
i'm going to pull this new code into a new linked partial part.  we just won't include that in cocoa.